src/invidious/views/template.ecr

```diff
@@ -149,7 +149,22 @@
                             <i class="icon ion-ios-wallet"></i>
                             <a href="https://invidious.io/donate/"><%= translate(locale, "footer_donate_page") %></a>
                         </span>
-                        <span><%= translate(locale, "Current version: ") %> <%= CURRENT_VERSION %>-<%= CURRENT_COMMIT %> @ <%= CURRENT_BRANCH %></span>
+                        <span>
+                            <%= translate(locale, "Current version: ") %>
+                            <% if CONFIG.modified_source_code_url %>
+                                <a href="<%= CONFIG.modified_source_code_url %>/commit/<%= CURRENT_COMMIT %>"><%= CURRENT_VERSION %>-<%= CURRENT_COMMIT %></a>
```

Comment:
How should we handle the case when the source code is hosted somewhere that doesn't adhere to this path structure?